Cargo.toml

```diff
@@ -12,7 +12,7 @@ members = ["contracts/*", "packages/*"]
 resolver = "2"
 
 [workspace.package]
-version = "0.16.1"
```

Comment:
0.16.0? or x.1?

Reply:
Yeah the version are inconsistent

Reply:
0.16.0, I published 0.16.1 and 0.16.2 that were not working so I yanked them

Reply:
I can still have a 0.16.0 version of utils so that's more consistent

Reply:
Okey but looking at the changes the utils now have version 0.16.0 while the other packages depend on 0.16.1. 

Just want to be consistent for doing a release so either we specify the version explicitly in utils (instead of using the workspace version) or do it your way (0.16.0 for all) 

Imo the versions also don't have to align with the abstract versions going forward.